### ISSUE TITLE
See below

### ISSUE THREAD

**Issue by Dennis5867** [qgis/QGIS#47437]
## User Feedback

I was on the opening screen and with a left click opned the folder for a project that was listed there. I then deleted three projects from the folderwhich resulted in QGIS entering a non-response state. I closed QGIS, reinstated the deleted project files and removed the project files from the folder where they were kept. I restarted QGIS and all seems well.

## Report Details

**Crash ID**: [84c538bf910ab6e6e0de1c26394b7d8ad1617c6a](https://github.com/qgis/QGIS/search?q=84c538bf910ab6e6e0de1c26394b7d8ad1617c6a&type=Issues)

**Stack Trace**
<pre>
QMenu::actionGeometry :
QMenu::mouseReleaseEvent :
QWidget::event :
QMenu::event :
QApplicationPrivate::notify_helper :
QApplication::notify :
QgsApplication::notify :
QCoreApplication::notifyInternal2 :
QApplicationPrivate::sendMouseEvent :
QSizePolicy::QSizePolicy :
QSizePolicy::QSizePolicy :
QApplicationPrivate::notify_helper :
QApplication::notify :
QgsApplication::notify :
QCoreApplication::notifyInternal2 :
QGuiApplicationPrivate::processMouseEvent :
QWindowSystemInterface::sendWindowSystemEvents :
QEventDispatcherWin32::processEvents :
qt_plugin_query_metadata :
QEventLoop::exec :
QCoreApplication::exec :
main :
BaseThreadInitThunk :
RtlUserThreadStart :
</pre>

**QGIS Info**
QGIS Version: 3.16.16-Hannover
QGIS code revision: f5778a89
Compiled against Qt: 5.15.2
Running against Qt: 5.15.2
Compiled against GDAL: 3.4.1
Running against GDAL: 3.4.1

**System Info**
CPU Type: x86_64
Kernel Type: winnt
Kernel Version: 10.0.22000

### Steps to reproduce the issue

None tried see notes above.

### Versions

QGIS Version: 3.16.16-Hannover
QGIS code revision: f5778a89
Compiled against Qt: 5.15.2
Running against Qt: 5.15.2
Compiled against GDAL: 3.4.1
Running against GDAL: 3.4.1

### Supported QGIS version

- [ ] I'm running a supported QGIS version according to the roadmap.

### New profile

- [ ] I tried with a new QGIS profile

**Comment by gioman** [comment]
@Dennis5867 

In general please:

1) use meaningful titles

2) always test with a new QGIS profile

3) if the problem persist, detail the steps to replicate attaching sample projects/data if necessary

> I restarted QGIS and all seems well.

So is the problem replicable somehow?

**Comment by Dennis5867** [comment]
My notes included a description of the steps I took that created the problem. While QGIS was open, I clicked on a project and sked that it open the folder rather than opening the project. In the folder, I deleted three project files. When I went back to QGIS the program froze.

**Comment by Dennis5867** [comment]
Giovanni,

In addition to my earlier reply, I thought of two more things that could be related:

When I saw that QGIS was not responding for over a minute, I tried to close the app using the “X” in the upper right corner. This could have generated the UH OH screen.

Subsequently, I was closing some of my files and got the not responding situation again. I looked a littloke more closely and noticed that the attributes file was open and it was counting through features. As a result, I let it run its course which took two to three minutes and then it was ok. It is possible a similar situation happened with issue 47437. From my perspective, since in neither case was any functional tasks being done, this may not be worth any more effort. However, I do want to recognize the diligence and good work that is clearly behind QGIS.

 

Thank you and others who are developing this incredible resource.

Dennis
